torchci/pages/metrics.tsx

```diff
@@ -121,9 +128,6 @@ function MasterCommitRedPanel({ params }: { params: RocksetParam[] }) {
       {
         type: "bar",
         stack: "all",
-        emphasis: {
```

Comment:
oops, I left in some random cleanups to the chart view. But I guess we can commit them as part of this